### PR TITLE
Code cleanup in TestBigQueryAvroConnectorTest

### PR DESCRIPTION
**Pull request by krvikash**
## Release notes

(X) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by raunaqmorarka — APPROVED** [review]

**Comment by krvikash** [comment]
(added comment and added the `@Override` notation)

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryAvroConnectorTest.java [hunk 1]
```diff
@@ -90,85 +86,9 @@ public void testSelectFailsForColumnName()
...
-                                results -> assertThat(results.getOnlyColumnAsSet()).isEqualTo(expectedResult.getOnlyColumnAsSet()));
-                    },
-                    results -> assertThat(results.getOnlyColumnAsSet()).isEqualTo(expectedResult.getOnlyColumnAsSet()));
-        }
-    }
-
-    @Override
-    @Test
-    public void testProjectionPushdownPhysicalInputSize()
+    protected Session withoutSmallFileThreshold(Session session)
```
Comment (marcinsbd) [inline plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryAvroConnectorTest.java:89]: is adding declaration of withoutSmallFileThreshold(Session session) is directly connected with removing those two other methods? if no pls split

Comment (krvikash) [reply plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryAvroConnectorTest.java:89]: Yes, It is connected. Both the test methods use `withoutSmallFileThreshold`.

**Review by vlad-lyutenko — APPROVED** [review]

**Comment by krvikash** [comment]
@raunaqmorarka Could you please run this PR against secrets?

**Comment by krvikash** [comment]
> The CI workflow run with tests that require additional secrets has been started: https://github.com/trinodb/trino/actions/runs/11685621094

CI Passed.

**Pull request merged**